test/micro/org/openjdk/bench/java/math/Fp16ConversionBenchmark.java

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
Can you update the copyright comment to be consistent with other source, and also correct the year to be only 2022? e.g. copy that from `Binary16Conversion.java`.